### PR TITLE
feat: add gradient start/end to projects table

### LINKED ISSUE #2926: [persistence] add project color and plumb through to the UI

### PR DESCRIPTION
**Pull request by mikeldking**
resolves #2926 

MVP of wiring up project gradient colors. This is done just to avoid a migration in the future.

Screenshot 2024-04-23 at 12 28 09 PM

### CODE REVIEW

File: src/phoenix/db/models.py [hunk 1]
```diff
@@ -93,6 +94,15 @@ class Project(Base):
     id: Mapped[int] = mapped_column(primary_key=True)
     name: Mapped[str]
     description: Mapped[Optional[str]]
+    gradient_start_color: Mapped[str] = mapped_column(
+        String,
+        server_default=func.text("'#5bdbff'"),
```
Comment (RogerHYang) [inline src/phoenix/db/models.py:99]: `text` needs to come from `sqlalchemy`, not `func`
```suggestion
        server_default=text("'#5bdbff'"),
```

File: src/phoenix/db/models.py [hunk 2]
```diff
@@ -93,6 +94,15 @@ class Project(Base):
...
+    )
+
+    gradient_end_color: Mapped[str] = mapped_column(
+        String,
+        server_default=func.text("'#1c76fc'"),
```
Comment (RogerHYang) [inline src/phoenix/db/models.py:104]: `text` needs to come from `sqlalchemy`, not `func`
```suggestion
        server_default=text("'#1c76fc'"),
```

**Review by RogerHYang — APPROVED** [review]

File: src/phoenix/db/models.py (hunk 1 above)
Comment (mikeldking) [reply src/phoenix/db/models.py:99]: Oh cool, will do

File: src/phoenix/db/models.py (hunk 2 above)
Comment (mikeldking) [reply src/phoenix/db/models.py]: Good catch. Slop on my part

**Pull request merged**